TAA/TAA.md

```diff
@@ -0,0 +1,141 @@
+Governance Framework (including the ​[Sovrin Ledger Access Policies​](https://sovrin.org/wp-content/uploads/Sovrin-Ledger-Access-Policies-V1.pdf)).
+* **f.** "**Personal Data**" means information that relates, directly or indirectly, to a data subject, including without limitation, names, email addresses, postal addresses, identification numbers, location data, online identifiers or one or more factors specific to the physical, physiological, genetic, mental, economic, cultural or social identity of the data subject.
+* **g.** "**Process**" or "**Processing**"​ means any operation or set of operations which is
+performed on Transactions data, whether or not by automated means, such as the access, collection, use, storage, disclosure, dissemination, combination, recording, organization, structuring, adaption, alteration, copying, transfer, retrieval, consultation, disposal, restriction, erasure and/or destruction of Transactions data.
+* **h.** "**Sovrin Governance Framework**" means the Sovrin Foundation’s governance policies and rules available at [​https://sovrin.org/governance-framework/](https://sovrin.org/governance-framework/) or any successor website.
+
+Unless otherwise defined above, all capitalized terms used in this Agreement shall have the meanings given to them in this Agreement or in the ​Sovrin Governance Framework and corresponding ​Sovrin Glossary​. The Sovrin Governance Framework and Sovrin Glossary is incorporated into this Agreement by reference only for purposes of use of such defined terms.
+
+## **2)Permission to Write to the Sovrin Ledger Networks**
+* a. The Sovrin Foundation hereby grants to Transaction Author a non-exclusive, non-assignable, non-sublicensable, royalty free, revocable license to write to and use the Sovrin Ledger Networks in accordance with this Agreement and the Sovrin Governance Framework.
+* b. When authoring Transactions under the policy of Permissioned Write Access, a Transaction Author may only write to the Sovrin Ledger Networks by using an authorized Transaction Endorser. In the event that Sovrin Foundation enables Public Write Access to the Sovrin Ledger Networks, the Transaction Author will not need a Transaction Endorser to endorse a Transaction.
+* c. Once an initial Transaction has been written to one of the Sovrin Ledger Networks by the Transaction Author ("**Initial Transaction**"), the Transaction Author is granted permission to make additional Transactions to update the state of a previous Transaction ("**Update Transactions**"). Please note, an Update Transaction does not remove the Initial Transaction, which will remain on the Sovrin Ledger due to its immutability and may remain on Sovrin Test Networks unless they are reset. Transaction Author may make Update Transactions if and only if the Transaction Author was the author of the Initial Transaction. Update Transactions are Transactions and are subject to all the terms of this Agreement.
+## **3)Transaction Author Obligations**
```

Comment:
Add whitespace between this and the previous section.

Reply:
agreed.